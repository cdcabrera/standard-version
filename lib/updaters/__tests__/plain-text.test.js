const { readVersion, writeVersion } = require('../types/plain-text');

describe('plain-text updaters', () => {
  it('should read version information', () => {
    const contents = {
      version: 'lorem ipsum'
    };

    expect(readVersion(JSON.stringify(contents))).toMatchSnapshot('read');
  });

  it('should return version information to write', () => {
    const contents = {
      version: 'lorem ipsum'
    };

    expect(writeVersion(JSON.stringify(contents), 'dolor sit')).toMatchSnapshot('write');
  });
});
