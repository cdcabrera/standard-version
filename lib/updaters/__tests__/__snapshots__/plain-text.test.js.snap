// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plain-text updaters should read version information: read 1`] = `"{\\"version\\":\\"lorem ipsum\\"}"`;

exports[`plain-text updaters should return version information to write: write 1`] = `"dolor sit"`;
